rundeckapp/grails-spa/packages/ui/src/components/activity/activityFilter.vue

```diff
@@ -170,8 +170,8 @@
       </div>
       <template slot="footer">
         <btn @click="filterOpen=false">{{$t('cancel')}}</btn>
-        <btn @click="search" class="btn btn-cta">{{$t('search')}}</btn>
-        <btn @click="saveFilter" type="success" class="pull-right">
+        <btn @click="search" type="cta" class="btn btn-cta">{{$t('search')}}</btn>
+        <btn @click="saveFilter" type="primary" class="btn-primary pull-right">
```

Comment:
This should be button default: Its not the primary action of the search feature.